service/src/main/java/org/cbioportal/service/AlterationCountService.java

```diff
@@ -14,64 +14,126 @@
                                                           boolean includeMissingAlterationsFromGenePanel,
                                                           Select<MutationEventType> mutationEventTypes,
                                                           Select<CNA> cnaEventTypes,
-                                                          QueryElement searchFusions);
+                                                          QueryElement searchFusions,
```

Comment:
Can you use the `AlterationFilter` object that you made rather than passing all these fields? This comment applies for all services, repositories, and mappers that have this pattern.

Reply:
I have collapsed all filter options in a single AlterationFilter object. This was a quite substantial refactoring. A lot of files changes. Can you re-review this?